components/squad/wikis/rainbowsix/squad_custom.lua

```diff
@@ -0,0 +1,75 @@
+---
+-- @Liquipedia
+-- wiki=rainbowsix
+-- page=Module:Squad/Custom
+--
+-- Please see https://github.com/Liquipedia/Lua-Modules to contribute
+--
+
+local Squad = require('Module:Squad')
+local SquadRow = require('Module:Squad/Row')
+local SquadAutoRefs = require('Module:SquadAuto/References')
+local Table = require('Module:Table')
+
+local CustomSquad = {}
+
+function CustomSquad.run(frame)
+	-- This function needs to be implemented on the local wiki is using manual Squad Tables
+end
```

Comment:
this comment imo does not make much sense in a /Custom on a wiki

Reply:
Changed to an error